test/densities/collider_setup/config.yaml

```diff
@@ -0,0 +1,35 @@
+# SMASH-1.5
+# Branch   : HEAD
+# System   : Linux-4.4.0-151-generic
+# Compiler : GNU 5.4.0
+# Build    : RelWithDebInfo
+# Date     : 2020-04-16T11:34:20
```

Comment:
Looks like you copied the output config, so better remove these lines and adjust the formatting of the rest to have the 4 character indentation.